src/Plugin/EntityMetaRelation/ListPage.php

```diff
@@ -230,12 +230,25 @@ public function build(array $form, FormStateInterface $form_state, ContentEntity
     // Get available filters.
     if ($list_source && $available_filters = $list_source->getAvailableFilters()) {
       $configuration = $this->getExposedFiltersConfiguration($list_source, $entity_meta_wrapper);
+
+      // Override checkbox.
+      $form[$key]['bundle_wrapper']['exposed_filters_wrapper']['exposed_filters_override'] = [
+        '#type' => 'checkbox',
+        '#title' => $this->t('Override default exposed filters'),
+        '#default_value' => isset($configuration),
```

Comment:
Let's not have `getExposedFiltersConfiguration()` return NILL and work out the override that means no filters somehow else. Maybe keep anther config option that says there is no override. And an empty array would mean none of the filters should be used. What do you think?